src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="search_bar"
+      data-qa="big"
+    >
+      <label class="search_label">
+        <input
+          class="bar_inpute big_inpute"
```

Comment:
If you use a modifier - do like this
```suggestion
          class="input input--big"
```
Apply all over the document

![208320250-cd1ba999-8eb7-40aa-9083-a9835fed2fc8](https://user-images.githubusercontent.com/117587463/208692121-245af94c-4006-42bd-a155-34379ffef5e8.png)